warp10/src/main/java/io/warp10/script/functions/ECRECOVER.java

```diff
@@ -46,6 +46,8 @@
 
   private static final String KEY_R = "r";
   private static final String KEY_S = "s";
+  private static final String KEY_I = "i";
```

Comment:
Why don't you call that parameter `h`?

Reply:
h is the actual cofactor of the curve, it is a fixed value, i is an index between 0 and the cofactor.